### DISCUSSION TITLE
add global menu

### DISCUSSION THREAD

**Discussion by SimoneRiccardi** [Alexays/Waybar#1933] [Ideas]
It would be nice to have a global menu of the used application on the status bar like in macos or kde.

It looks nicer and It's more space efficient without giving up features ( because you can remove the menu bar on top of all applications on the screen and show the menu options just of the application that you're using on the status bar)

**Comment by AndyP3r3z** [comment]
For what I see, [it's an issue under discussion](https://github.com/Alexays/Waybar/issues/288).
